gluoncv/utils/metrics/coco_instance.py

```diff
@@ -23,49 +24,60 @@ class COCOInstanceMetric(mx.metric.EvalMetric):
         Prefix for the saved JSON results.
     use_time : bool
         Append unique datetime string to created JSON file name if ``True``.
-    cleanup : bool
-        Remove created JSON file if ``True``.
+    dump_json : bool
+        Dump evaluation JSON file if ``True``. When use_ext is true, evaluation is always dumped
+    use_ext : bool
+        Whether to use external module built by NVIDIA, which is significantly faster.
```

Comment:
any way to check whether if't nvidia's version?

Reply:
I'm not sure it's a fork of the original mscoco api